src/app/hooks/useViewTracker/index.test.tsx

```diff
@@ -43,7 +43,8 @@ const IntersectionObserver = jest.fn(cb => {
 
 const getObserverInstance = (element: HTMLElement) => {
   try {
-    // @ts-expect-error required for testing purposes
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
```

Comment:
Shall we add the error as a note (in the same way we would for @ts-expect-error)